client/src/components/PaymentMethods/paymentMethods.tsx

```diff
@@ -0,0 +1,37 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { Redirect } from 'react-router-dom';
+import PaymentMethod from '../../helpers/APICalls/paymentMethods';
+import SettingHeader from '../SettingsHeader/SettingsHeader';
+
+const PaymentMethods = (): JSX.Element => {
+  const createPaymentProfile = async () => {
+    console.log('Am called');
+
+    const result = await PaymentMethod();
+
+    console.log(result);
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!